src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Random random = new Random();
```

Comment:
It is better to put the random in the class field, like: 
![image](https://user-images.githubusercontent.com/12864128/208109992-6e55eefe-21cd-4cdc-b2fe-d5f8de00f3c3.png)

Reply:
You didn't correct this comment.